test/bandit/server_test.exs

```diff
@@ -25,7 +25,7 @@ defmodule ServerTest do
   end
 
   test "server logs connection error detail log at startup" do
-    pid = start_supervised!({Bandit, scheme: :http, plug: __MODULE__})
+    pid = start_supervised!({Bandit, scheme: :http, plug: __MODULE__, port: 40_000})
```

Comment:
You can pass a value of `0` for port and it will [pick an open port automatically](https://hexdocs.pm/thousand_island/ThousandIsland.html#t:options/0).

Reply:
Oh, cool, like `gen_tcp` :clap:

Reply:
Yup, exactly. Bandit is just a thin layer on Thousand Island which is just a thin layer on gen_tcp. It's all transparent